homeassistant/components/switchmate/switch.py

```diff
@@ -49,7 +49,7 @@ def __init__(self, mac, name, flip_on_off) -> None:
 
         self._mac = mac
         self._name = name
-        self._device = switchmate.Switchmate(mac=mac, flip_on_off=flip_on_off)
+        self._device = Switchmate(mac=mac, flip_on_off=flip_on_off)
```

Comment:
Bleak doesn't use mac addresses on all platforms so at some point this should switch to be an address instead